.travis.yml

```diff
@@ -1,8 +1,9 @@
-language: node_js
-node_js: "8"
+language: java
```

Comment:
these modifications are here to workaround a JVm bug affecting the 11.0.2 which is used by default by Travis